### PR TITLE
feat: add marking for LTS releases

### LINKED ISSUE #10508: Add marking for LTS releases
In the [Release Notes](https://docs.wagtail.org/en/stable/releases/index.html) section, add a marker (can be something as simple as the one shown in the screenshot) to show which releases are LTS
image

### PR DESCRIPTION
**Pull request by souzathg**
Fixes #10508

Adds marking to LTS releases in the docs, as requested in this issue.
I added the markings by changing the title of the files. 
Please don't hesitate to suggest any changes.

_Please check the following:_

-   [X] For new features: Has the documentation been updated accordingly?

### CODE REVIEW

**Review by gasman — CHANGES_REQUESTED** [review]
Thanks @souzathg!

File: docs/releases/0.8.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 =========================
```
[the remark this answers is not in the document]
Comment (souzathg) [reply docs/releases/0.8.rst:1]: Ah, ok! Thanks for pointing that out!
As soon as it's done, I'll reply again.

[the remark this answers is not in the document]
Comment (souzathg) [reply docs/releases/0.8.rst]: I increased the length of the lines, as requested 😄

**Review by gasman — APPROVED** [review]
Thanks - all looks good to me!

**Comment by gasman** [comment]
Merged in ce620bf57b66576fe4c0cc1ed58e1a0f6e9fbeb8.

**Pull request closed without merging**